packages/@webex/plugin-meetings/src/media/index.ts

```
@@ -182,6 +185,12 @@ Media.createMediaConnection = (
       config.disableAudioMainDtx = disableAudioMainDtx;
     }
 
+    if (BrowserInfo.isFirefox()) {
+      config.doFullIce = true;
+
+      config.stopIceGatheringAfterFirstRelayCandidate = stopIceGatheringAfterFirstRelayCandidate;
+    }
```

A few questions:

1. What happens if I configure `stopIceGatheringAfterFirstRelayCandidate = true` and the browser isn't Firefox? Your unit tests imply that the option would be ignored, but there's no way an SDK user would know this. If we want to only apply this option to Firefox, it should be clear to the SDK user, e.g. by including "Firefox" in the option name.

2. What happens if `doFullIce = true` is false but `stopIceGatheringAfterFirstRelayCandidate = true`? I believe WCME would ignore the second option, but again this is not made clear to the SDK user. Perhaps we should throw an error/warning either at the SDK level or the WCME level?

3. Do we really want to condition for Firefox here? Should we let the web client be in full control of deciding when to enable/disable these options?

---

1. I think it would be enough to just explain in the config comments that the option currently applies only to FF. Leaving out the Firefox browser name from the config option allows us more flexibility in case we want to use this for other browsers in the future too (unlikely, but you never know...)
2. Such combination currently doesn't make sense, because we don't wait for gathering of candidates when doing ice-lite, but in theory we could do this in the future, so for now I don't mind we could throw an error that such combination is not supported, but I'm fine also with just silently ignoring `stopIceGatheringAfterFirstRelayCandidate` in that case.
3. No, we definitely don't want the app to be controlling which browsers do full ICE, because our backend really requires FF to have full ICE and not sure if it would work with other browsers. Other clients of the JS-SDK won't know which browsers should have full ICE and which shouldn't, so the logic of enabling full ice for FF and only for FF needs to be somewwhere in JS-SDK or layers below.